### PR TITLE
Readme Update (Project Description & Contributors)

### PR DESCRIPTION
**Pull request by chadbaum**
Fixed up the marketing of our repository so that it looks nice for visitors to the repo.

- Put coveralls and codeship indicators side-by-side
- Added brief description of the application
- Added planned features list
- Added gems we are currently using
- Reformatted contributors:  your name is now your email link, I added Zach's GitHub, and made it all look pretty.  If you have a website you want to add, or a LinkedIn, just let me know.

Enjoy.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,34 @@
-[![Coverage Status](https://coveralls.io/repos/github/TeamStoic/StoicChess/badge.svg?branch=master)](https://coveralls.io/github/TeamStoic/StoicChess?branch=master)
+[![Codeship Status for TeamStoic/StoicChess](https://codeship.com/projects/8443e920-1f83-0134-0385-3ae309bf0e43/status?branch=master)](https://codeship.com/projects/160373) [![Coverage Status](https://coveralls.io/repos/github/TeamStoic/StoicChess/badge.svg?branch=master)](https://coveralls.io/github/TeamStoic/StoicChess?branch=master)
 
-[ ![Codeship Status for TeamStoic/StoicChess](https://codeship.com/projects/8443e920-1f83-0134-0385-3ae309bf0e43/status?branch=master)](https://codeship.com/projects/160373)
 
 
-== Contributors
+StoicChess is a chess web application built on Ruby on Rails by an Agile team of 5 developers as a capstone project for [The Firehose Project](http://www.thefirehoseproject.com).  The business logic of the application includes enforcing all of the rules of chess, including: moves, obstruction, capture, promotion, castling, en passant, check, and checkmate.  
 
+**Planned Features**
```
Comment (lasley) [inline README.md]: Should use a header for these instead of bold:

```
Planned Features
=============
```

File: README.md [hunk 2]
```diff
@@ -1,18 +1,34 @@
-[![Coverage Status](https://coveralls.io/repos/github/TeamStoic/StoicChess/badge.svg?branch=master)](https://coveralls.io/github/TeamStoic/StoicChess?branch=master)
+[![Codeship Status for TeamStoic/StoicChess](https://codeship.com/projects/8443e920-1f83-0134-0385-3ae309bf0e43/status?branch=master)](https://codeship.com/projects/160373) [![Coverage Status](https://coveralls.io/repos/github/TeamStoic/StoicChess/badge.svg?branch=master)](https://coveralls.io/github/TeamStoic/StoicChess?branch=master)
```
Comment (lasley) [inline README.md]: Add a Title:
```

==============
{module_title}
==============
```

File: README.md [hunk 3]
```diff
@@ -1,18 +1,34 @@
...
+**Planned Features**
+* Drag and drop interaction with pieces
+* WebSockets to provide multi-user support for playing head-to-head against another human opponent without having to refresh the page
+* Forfeit and draw options
+* Lobby for finding opponents and one-to-one chat with opponent
+* Rating system for tracking wins/losses
+* Facebook authorization
 
-* Zach Berman - zberms@gmail.com
+**Gems**
```
Comment (lasley) [inline README.md]:
```
Gems
====
```

File: README.md [hunk 4]
```diff
@@ -1,18 +1,34 @@
...
+* RSpec
+* Capybara
+* Coveralls
+* FactoryGirl
 
-* Aleks Gorbenko - AleksEdGorbenko@gmail.com
-..[Personal Blog](https://aleksgorbenko.com)
-..[GitHub](https://github.com/aleksgorbenko)
-..[LinkedIn](https://uk.linkedin.com/in/aleks-gorbenko-web-developer)
+**Contributors**
```
Comment (lasley) [inline README.md]:
```
Contributors
===========
```

File: README.md [hunk 5]
```diff
@@ -1,18 +1,34 @@
...
 
-* Ethan He - ethanhe.dev@gmail.com,[GitHub](https://github.com/Se7enB2st) 
+* [Chad Baum](mailto:chadbaum@gmail.com) - [Github](https://github.com/chadbaum) | [LinkedIn](https://linkedin.com/in/chadbaum)
 
-* George Fourikis - https://github.com/GeorgeFourikis - georgefourikis@hotmail.gr
+* [Zach Berman](mailto:zberms@gmail.com) - [GitHub](https://github.com/ZachBerman)
+
+* [George Fourikis](mailto:georgefourikis@hotmail.gr) - [GitHub](https://github.com/GeorgeFourikis)
+
+* [Aleks Gorbenko](mailto:aleksedgorbenko@gmail.com) - [GitHub](https://github.com/aleksgorbenko) | [LinkedIn](https://uk.linkedin.com/in/aleks-gorbenko-web-developer) | [Website](https://aleksgorbenko.com)
```
Comment (lasley) [inline README.md]: Chose either `-` or `|` separation, not both.

**Comment by lasley** [comment]
Here's the template I typically use. It's an RST, so will need slight adaptation. Here's a live sample - https://raw.githubusercontent.com/laslabs/Python-Carepoint/release/0.1/README.md

File: README.md [hunk 6]
```diff
@@ -1,18 +1,39 @@
-[![Coverage Status](https://coveralls.io/repos/github/TeamStoic/StoicChess/badge.svg?branch=master)](https://coveralls.io/github/TeamStoic/StoicChess?branch=master)
+==========
+StoicChess
+==========
 
-[ ![Codeship Status for TeamStoic/StoicChess](https://codeship.com/projects/8443e920-1f83-0134-0385-3ae309bf0e43/status?branch=master)](https://codeship.com/projects/160373)
+[![Codeship Status for TeamStoic/StoicChess](https://codeship.com/projects/8443e920-1f83-0134-0385-3ae309bf0e43/status?branch=master)](https://codeship.com/projects/160373) [![Coverage Status](https://coveralls.io/repos/github/TeamStoic/StoicChess/badge.svg?branch=master)](https://coveralls.io/github/TeamStoic/StoicChess?branch=master)
```
Comment (lasley) [inline README.md]: IMO the badges should be Above the main title. Also, we should think about what license this is to be released under & add a badge for that as well

**Comment by aleksgorbenko** [comment]
Hey - looks neat. Could you add my portfolio site as well, please --> https://aleksgorbenko.github.io/

**Comment by chadbaum** [comment]
Done

**Comment by aleksgorbenko** [comment]
👍

**Comment by ZachBerman** [comment]
Chad can you add mine as well ---> [http://zachberman.github.io/](url)
                                                           [https://ca.linkedin.com/in/zachbdev](url)

**Comment by ZachBerman** [comment]
Looks good - although features may change in the end....

**Pull request merged**